Makefile

```diff
@@ -58,10 +58,16 @@ test:
 
 load-fixtures:
 	docker-compose up -d db
-	echo "Wait for db to be ready..."
+	echo "Wait for database to be ready..."
 	sleep 5
-	docker-compose exec db psql "postgres://postgres:toor@db:5432/bcda?sslmode=disable" -f /var/db/fixtures.sql
-	docker-compose exec db psql "postgres://postgres:toor@db:5432/bcda?sslmode=disable" -f /var/db/synthetic_cclf_files_beneficiaries.sql
+	docker-compose run db psql "postgres://postgres:toor@db:5432/bcda?sslmode=disable" -f /var/db/fixtures.sql
```

Comment:
Instead of adding to fixtures, what about working in a sql-migrate execution like in unit_test.sh?  I think that is more in line with what we are doing in production since fixtures.sql isn't run there afaik.

Reply:
I think this is getting into the area of DB Migration strategy, which remains a topic of discussion for the whole team.